### ISSUE TITLE
AssemblyScript Public Meeting #23 - October 21st, 2020

### ISSUE THREAD

**Issue by torch2424** [AssemblyScript/working-group#48]
# Date and Time

This public meeting will take place: October 21st, 2020, 18:00 UTC (11:00 AM US PDT, UTC -8)

# General Agenda

* Agenda Items from comments left on this Github issue
* Additional in-meeting comments / discussion
* [If time allows] Recap of [WebAssembly CG meeting](https://github.com/WebAssembly/meetings) if anyone attended

Feel free to comment on this issue if you have any agenda items you would like to bring up. 
Meeting Notes will be placed on this issue for those who cannot make the meeting.

# Meeting Information

We will use the AssemblyScript discord for meetings: https://discord.gg/vSBbJ5T . We will meet in the "meeting" voice channel :+1: 

Anyone who is contributing to the AssemblyScript project, building something with AssemblyScript, interested in the Assembly project or WebAssembly in general, is welcome to join! 😄

Notes will be taken by the host, and posted after the meeting. Notes are free to be edited through comments on the meeting notes at a later time.

**Comment by torch2424** [comment]
# Meeting Notes

* @saulecabrera : Closures with Transforms:
   * Looking into simpler closures using transforms? Asking if it is possible as Saul has experience with languages with Macros where this is possible
   * @dcodeIO  : Since there are already errors for closures, the transform wouldn’t be able to get around the error, and the transform itself would be really difficult.
* @saulecabrera : Nullish Coalescing PR
   * https://github.com/AssemblyScript/assemblyscript/pull/884 
   * What does it need to get merged?
   * @dcodeIO : Essentially it’s nested properties, so you’d have to break out of a block when your compiling another expression. 
* @jayphelps : Status of GC and Binaryen?
   * Seeing stuff for Arrays and structs and stuff
   * @dcodeIO : Getting a lot done, waiting on others to finish up their work, and then we can get across the finish like for things like new structs, and RTT. 
      * https://github.com/WebAssembly/binaryen/issues/3063 
      * https://github.com/WebAssembly/binaryen/blob/master/src/wasm-type.h#L28 
* Wasm Community Group Updates
* @battlelinegames : What WasmGC?
   * @dcodeIO  & @jayphelps : TL;DR If everyone language had to ship down their own GC, it’d greatly inflate the wasm module size, because good GC requires a lot of code. Providing a primitive in runtimes makes module sizes wayyyyyy smaller, and faster (because the GC is running in native)
   * The fundamental conflict in the spec is: Starting small enough where we can build upon, but, not so small, that we can’t do anything with it, or pin us on a foundation that wont work long term.
* Super secret stuff (no notes)
   * @torch2424 : Haha! :’) (We're kidding, was just some friendly chat)

**Comment by torch2424** [comment]
Closing this in favor of the next meeting #49

Feel free to continue discussions or ask questions here :smile:
